test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -41,10 +68,12 @@
     ( #caseid, caseid[0], caseid[1], clips1, -1, area1, clips2, -1, area2, \
                 clips3, -1, area1 + area2, settings)
 
-template <typename P>
+template <typename P, bool ClockWise>
```

Comment:
I added `ccw` tests for difference, it was not earlier there. Most cases work, but a few of them don't - they are now conditionally